specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/privateEndpoint.json

```diff
@@ -303,6 +303,87 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/locations/{location}/availablePrivateEndpointTypes": {
+      "get": {
+        "operationId": "AvailablePrivateEndpointTypes_List",
```

Comment:
SDK generator would use `operationId` to get class and method names (e.g. `AvailablePrivateEndpointTypes.List()`). Since this method is related to privateEndpoints, maybe it would be better to name it `PrivateEndpoints_ListAvailablePrivateEndpointTypes` or `PrivateEndpoints_ListAvailableTypes`

Reply:
I agree with. Previously I was thinking the cmdlet is under different path from private endpoint /private link service, so they should be in different operation class id. I have already fix it, this make code looks more neat. Thanks.